### ISSUE TITLE
Weird results

### ISSUE THREAD

**Issue by pydima** [scylladb/diskplorer#29]
Hi, thanks a lot for developing this tool, I think that's a fantastic idea. I've tried to run it but got really weird results genereated (screen attached). Any ideas on how to better debug this or why I've got this result? Thanks.
![Screenshot from 2022-11-08 08-54-54](https://user-images.githubusercontent.com/5312367/200521572-c59254d0-db60-4160-b988-0b061ff6b78e.png)

**Comment by avikivity** [comment]
What kind of disk are you testing?

If it's a rotating disk, maybe the tool estimated read iops incorrectly. You can try with --max-read-iops 200 (adjust according to what you think the disk can bear).

**Comment by avikivity** [comment]
Actually, it looks like SSD since you got 2 GB/s. What's the make and model of the disk?

Please upload the results directory with the json files somewhere.

**Comment by pydima** [comment]
Yes, it's SSD disk from Intel - https://ark.intel.com/content/www/us/en/ark/products/series/122570/intel-ssd-dc-p4510-series.html. I tried to run diskplorer with default iops/write bandwith (auto-discover one) and with explicitly providing the values from the disk spec - the result is the same. Here is the link to result file: https://drive.google.com/file/d/1dEASw8X7Z0Xny1xWo8xWM7ZHMUGsq_5d/view?usp=sharing

**Comment by avikivity** [comment]
```json
      "jobname": "job(r_idx=1,w_idx=0,write_bw=0,r_iops=31825)",
      "groupid": 0,
      "error": 0,
      "eta": 0,
      "elapsed": 37,
      "job options": {
        "rw": "randread",
        "bs": "4096",
        "iodepth": "1000",
        "rate_iops": "397",
        "rate_process": "poisson"
      },
      "read": {
        "io_bytes": 3534966784,
        "io_kbytes": 3452116,
        "bw_bytes": 117761569,
        "bw": 115001,
        "iops": 28748.017856,
        "runtime": 30018,
```

We commanded 80*397=31827 (do you have 80 threads?) and only got 28748.017856 iops. So diskplorer disqualified (diskualified?) this result.

Please pick the .fio directory corresponding to this job (r_idx=1,w_idx=0) and rerun it under fio directly, look at top/blktrace etc to see what went wrong.

**Comment by pydima** [comment]
There are 80 cores, so I think 80 threads should be fine. Thanks for the pointer, will try to run it!
